client/login/magic-login/handle-emailed-link-form.jsx

```diff
@@ -142,6 +109,9 @@ class HandleEmailedLinkForm extends React.Component {
 		return (
 			<EmptyContent
 				action={ action }
+				className={ classNames( {
+					'magic-login__isfetchingauth': isFetching,
```

Comment:
I believe this modifier class should be named `is-fetching-auth` (https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md)